content/docs/tutorials/getting-started-aws-letsencrypt/README.md

```diff
@@ -0,0 +1,654 @@
+
+```yaml file=../../../../public/docs/tutorials/getting-started-aws-letsencrypt/deployment.yaml
+```
+🔗 <a href="deployment.yaml">`deployment.yaml`</a>
+
+```bash
+kubectl apply -f deployment.yaml
+```
+
+You also need to create a Kubernetes LoadBalancer Service, so that connections from the Internet can be routed to the web server Pod.
+When you create the following Kubernetes Service, an AWS classic load balancer with an ephemeral public IP address will also be created:
+
+```yaml file=../../../../public/docs/tutorials/getting-started-aws-letsencrypt/service.yaml
+```
+🔗 <a href="service.yaml">`service.yaml`</a>
+
+```bash
+kubectl apply -f service.yaml
+```
+
+Within 2-3 minutes, a load balancer should have been provisioned with a public IP.
+
+```bash
+kubectl get service helloweb
+```
+
+Sample output
+
+```terminal
+$ kubectl get service helloweb
+NAME       TYPE           CLUSTER-IP       EXTERNAL-IP                                                              PORT(S)         AGE
+helloweb   LoadBalancer   10.100.175.247   ae25d292150aa4e3e90e6c25376f9a7d-496307726.us-west-2.elb.amazonaws.com   443:32184/TCP   6m
+```
+
+The `EXTERNAL-IP` will be different for you and it may be different each time you re-create the LoadBalancer service,
+but it will have a stable DNS host name associated with it.
+
+> ℹ️ By default, EKS creates [classic](https://aws.amazon.com/elasticloadbalancing/features/#Product_comparisons)
+> load balancers for LoadBalancer Services in the cluster,
+> using the [Legacy Cloud Provider Load balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html#lbc-legacy).
+> This is convenient for this tutorial because it does not require any additional software or configuration, but
+> [AWS Cloud Provider Load balancer Controller is legacy and is currently only receiving critical bug fixes](https://aws.github.io/aws-eks-best-practices/networking/loadbalancing/loadbalancing/#choosing-load-balancer-type)
+> according to the Amazon's [EKS Best Practices Guide](https://aws.github.io/aws-eks-best-practices/).
+> Consider using the [AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html) instead.
+
+The stable DNS host name of the load balancer can be used as an alias for the `www` record in your chosen `$DOMAIN_NAME`
+by creating a Route53 [Alias Record](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-choosing-alias-non-alias.html):
+
+```bash
+HOSTED_ZONE_ID=$(aws route53 list-hosted-zones-by-name --dns-name $DOMAIN_NAME --query "HostedZones[0].Id" --output text)
+ELB_CANONICAL_HOSTED_ZONE_NAME=$(kubectl get svc helloweb --output=jsonpath='{ .status.loadBalancer.ingress[0].hostname }')
+aws elb describe-load-balancers --query "LoadBalancerDescriptions[?CanonicalHostedZoneName == '$ELB_CANONICAL_HOSTED_ZONE_NAME'] | [0]" \
+| jq '{
+  "Comment": "Creating a CNAME record",
```

Comment:
suggestion: The reference to a CNAME in this comment might be a little jarring for newbies given the action references an `A` record and we're talking about alias records.

```suggestion
  "Comment": "Creating an alias record",
```